base_time_window/models/time_window_mixin.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import math
+from datetime import time
+
+from psycopg2.extensions import AsIs
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class TimeWindowMixin(models.AbstractModel):
+
+    _name = "time.window.mixin"
+    _description = "Time Window"
+    _order = "start"
+
+    # TODO patch api.constrains with field here?
+    _overlap_check_field = False
+
+    start = fields.Float("From", required=True)
```

Comment:
being a mixin this fields will land on another model: what about prefixing them properly?
Eg: `time_window_*` or `window_*`?